### DISCUSSION TITLE
"Launch Item" button in right side item editor when modifying item in M2M,M2A interface

### DISCUSSION THREAD

**Discussion by dwene** [directus/directus#6484] [Feature Requests]
This discussion started in the discord chat. 
![image](https://user-images.githubusercontent.com/12628251/123307745-7a0b6380-d4e8-11eb-9916-4798432daf7c.png)

When editing an item that's in a M2M or M2A interface, it would be nice to have a way to view the revisions and revert back unwanted changes. I often times hide some collections from clients that are are available to them via the M2A or M2M interface to remove confusion.

@benhaynes had the idea to add a button in that interface that would go to the individual item in the collection, and people could look at revisions and comments there.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
